docs/jazzy.yml

```diff
@@ -11,6 +11,14 @@ skip_undocumented: Yes
 hide_documentation_coverage: Yes
 
 custom_categories:
+  - name: Examples
+    children:
+      - Storyboards
+      - Advanced
+      - Basic
+      - Embedded-Navigation
+      - Styled-UI-Elements
+      - Waypoint-Arrival-Screen
   - name: Guides
```

Comment:
Probably makes sense to move “Guides” up above “Examples”. Eventually we’d add more conceptual content here that could serve as a prerequisite for the examples.